docs/sources/administration/provisioning.md

```diff
@@ -146,8 +146,10 @@ Since not all datasources have the same configuration settings we only have the
 | tlsAuth                 | boolean | _All_                                                            | Enable TLS authentication using client cert configured in secure json data                  |
 | tlsAuthWithCACert       | boolean | _All_                                                            | Enable TLS authentication using CA cert                                                     |
 | tlsSkipVerify           | boolean | _All_                                                            | Controls whether a client verifies the server's certificate chain and host name.            |
+| serverName              | string  | _All_                                                            | Optional. Controls the server name used for certificate common name/subject alternative name verification. Empty defaults to using the data
+source URL. |
```

Comment:
Could it be written "Defaults to using the data source URL" instead? Would make more sense to me than how it's currently written at least.